### DISCUSSION TITLE
Livewire version 3 dispatchTo event is not working when call livewire components

### DISCUSSION THREAD

**Discussion by jenishprajapati-kombee** [Power-Components/livewire-powergrid#1224] [Q&A]
Please find below screenshots for more details:

![image](https://github.com/Power-Components/livewire-powergrid/assets/67471303/6e4948ea-9d22-41e2-9b5b-7a5f0d88e30f)
![image](https://github.com/Power-Components/livewire-powergrid/assets/67471303/3e4aaa90-37dd-4ac9-ae32-3fd8614ed3a1)

**Comment by YnievesDotNet** [comment]
> Please find below screenshots for more details:
> 
> 
> 
> 
> 
> ![image](https://github.com/Power-Components/livewire-powergrid/assets/67471303/6e4948ea-9d22-41e2-9b5b-7a5f0d88e30f)
> 
> ![image](https://github.com/Power-Components/livewire-powergrid/assets/67471303/3e4aaa90-37dd-4ac9-ae32-3fd8614ed3a1)
> 
> 

I have same error. Don't dispatch the event to normal Livewire component

**Comment by YnievesDotNet** [comment]
I solved the problem with the following solution.

Are you calling the component that will listen to the event in the same view where you have inserted the PowerGrid table?

I solved it by adding it to the same view.

<img width="717" alt="Captura de pantalla 2024-02-04 a la(s) 11 01 00 a m" src="https://github.com/Power-Components/livewire-powergrid/assets/1451485/36d1aa4d-41e6-4429-845f-08514b57e4b9">

**Reply by jenishprajapati-kombee** [reply]
Everything's running smoothly now. Thanks!
